### PR TITLE
Feature/answer page

### PR DESCRIPTION
**Pull request by pengin7384**
Complete
- Connect with ResultPage
- Update UI

### CODE REVIEW

File: src/components/ui/AnswerBoard.js [hunk 1]
```diff
@@ -136,21 +131,25 @@ class AnswerBoard extends React.Component {
...
   render() {
-    const {questionNumber, userAnswer, language} = this.props;
-    const question = questions[questionNumber-1];
-    const originalAnswer = answers[questionNumber-1].Answer;
+    const {language} = this.props;
+    const {Id, Selected} = this.props.location.state;
+    const questionNumber = Id.toString();
+    const userAnswer = Selected.toString();
+    const question = questions[questionNumber - 1];
+    const originalAnswer = answers[questionNumber - 1].Answer;
```
Comment (minsulee2) [inline src/components/ui/AnswerBoard.js:140]: Those should be dealt with in constructor.
Can you move them into it?

File: src/components/ui/AnswerBoard.js [hunk 2]
```diff
@@ -175,31 +174,33 @@ class AnswerBoard extends React.Component {
...
                     <AnswerWrapper backgroundColor={buttonColor}>
-
-                      <BoxWrapper>
-                        <Box src={require('../../static/img/answer/CorrectWhite.svg')}/>
-
+                      <BoxWrapper flex={1}>
+                        <Box src={checkBox}/>
+                      </BoxWrapper>
+                      <Answer flex={6}>{question.Answers[i]}</Answer>
+                      <BoxWrapper flex={1}>
```
Comment (minsulee2) [inline src/components/ui/AnswerBoard.js:202]: Please make some space for one another

File: src/components/ui/AnswerBoard.js [hunk 3]
```diff
@@ -212,4 +213,4 @@ class AnswerBoard extends React.Component {
   }
 }
 
-export default AnswerBoard;
+export default withRouter(AnswerBoard);
```
Comment (minsulee2) [inline src/components/ui/AnswerBoard.js:216]: What makes it difference (My wonder)?

File: src/components/ui/AnswerBoard.js [hunk 4]
```diff
@@ -136,21 +131,25 @@ class AnswerBoard extends React.Component {
...
-    const originalAnswer = answers[questionNumber-1].Answer;
+    const {language} = this.props;
+    const {Id, Selected} = this.props.location.state;
+    const questionNumber = Id.toString();
+    const userAnswer = Selected.toString();
+    const question = questions[questionNumber - 1];
+    const originalAnswer = answers[questionNumber - 1].Answer;
     const correct = (userAnswer === originalAnswer);
 
     return (
```
Comment (minsulee2) [inline src/components/ui/AnswerBoard.js:143]: Can you make below here 4 spaces indentation??

File: src/components/ui/AnswerBoard.js (hunk 2 above)
Comment (pengin7384) [reply src/components/ui/AnswerBoard.js:202]: Done!

File: src/components/ui/AnswerBoard.js (hunk 3 above)
Comment (pengin7384) [reply src/components/ui/AnswerBoard.js:216]: `withRouter` is for `this.props.history.goBack()'! 
`goBack()` is only available when using `withRoute`

File: src/components/ui/AnswerBoard.js (hunk 4 above)
Comment (pengin7384) [reply src/components/ui/AnswerBoard.js:143]: Done! Thx

File: src/components/ui/AnswerBoard.js (hunk 1 above)
Comment (pengin7384) [reply src/components/ui/AnswerBoard.js:140]: Done!

**Comment by pengin7384** [comment]
@minsulee2 PTAL!

File: src/components/ui/AnswerBoard.js [hunk 5]
```diff
@@ -15,7 +14,29 @@ const Wrapper = styled.div`
...
+    padding-left: 10px;
+    padding-right: 10px;
+    margin-bottom: 10px;
+    border-radius: 4px;
+    outline: none;
+    font-family: IBM Plex Sans;
+    font-size: 14px;
+    font-weight: bold;
+    text-align: center;
+    color: #333333;
```
Comment (minsulee2) [inline src/components/ui/AnswerBoard.js:33]: It should be background-color: SOMETHING

File: src/components/ui/AnswerBoard.js [hunk 6]
```diff
@@ -15,7 +14,29 @@ const Wrapper = styled.div`
...
+    :hover {
```
Comment (minsulee2) [inline src/components/ui/AnswerBoard.js:34]: Remove hover please

File: src/components/ui/AnswerBoard.js (hunk 5 above)
Comment (pengin7384) [reply src/components/ui/AnswerBoard.js:33]: Done!

File: src/components/ui/AnswerBoard.js (hunk 6 above)
Comment (pengin7384) [reply src/components/ui/AnswerBoard.js:34]: Done!

**Pull request merged**